examples/saml-service-provider/src/main/java/example/armeria/server/saml/sp/Main.java

```diff
@@ -0,0 +1,99 @@
+package example.armeria.server.saml.sp;
+
+import static com.linecorp.armeria.server.saml.SamlEndpoint.ofHttpPost;
+
+import java.io.File;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+
+import org.opensaml.xmlsec.signature.support.SignatureConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.linecorp.armeria.server.Server;
+import com.linecorp.armeria.server.ServerBuilder;
+import com.linecorp.armeria.server.saml.KeyStoreCredentialResolverBuilder;
+import com.linecorp.armeria.server.saml.SamlServiceProvider;
+import com.linecorp.armeria.server.saml.SamlServiceProviderBuilder;
+
+public class Main {
+    private static final Logger logger = LoggerFactory.getLogger(Main.class);
+
+    /**
+     * Configures an identity provider with <a href="https://idp.ssocircle.com/meta-idp.xml">
+     * the metadata of the SSO Circle</a>. You must <a href="https://idp.ssocircle.com/sso/hos/SPMetaInter.jsp">
+     * register</a> this service provider, which we are configuring here, to the SSO Circle.
+     * You can get the metadata of this service provider from {@code https://localhost:8443/saml/metadata}
+     * after starting this server.
+     *
+     * <p>The {@code signing} key pair in the keystore {@code sample.jks} can be generated with the following
+     * command. You can also generate the {@code encryption} key pair with the same command, or you can use
+     * the same key pair for encryption.
+     * <pre>{@code
+     * $ keytool -genkeypair -keystore sample.jks -storepass 'N5^X[hvG' -keyalg rsa -sigalg sha1withrsa
+     *     -dname 'CN=Unknown, OU=Unknown, O=Unknown, L=Unknown, ST=Unknown, C=Unknown' -alias signing
```

Comment:
- How about showing two `keytool` commands that generate `signing` and `encryption` respectively although it's repetitive? It may simplify the explanation above:
  ```
  The {@code signing} and {@code encryption} key pairs in the keystore {@code sample.jks} can be
  generated with the following commands:
  ```